src/Validators/Validator.Strings.cs

```diff
@@ -75,11 +75,11 @@ internal static void ThrowIfNotEqualsIgnoreCase(string value, string paramName,
     }
 
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    internal static void ThrowIfEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string otherString)
+    internal static void ThrowIfEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string otherString, StringComparison comparisonType = StringComparison.Ordinal)
```

Comment:
If we add this defaulted to `Ordinal` in the already exposed method we can make this required and call only this method also for the `IfEqualsIgnoreCase` (removing the existing `ThrowIfEqualsIgnoreCase`) WDYT?
```suggestion
    internal static void ThrowIfEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string otherString, StringComparison comparisonType)
```